### DISCUSSION TITLE
With what does scv.utils.merge  merges adata and loom file?

### DISCUSSION THREAD

**Discussion by parkjooyoung99** [theislab/scvelo#878] [Q&A]
I wonder how 'scv.utils.merge' merges adata and loom file.  

after merging, there are no barcodes left. 
I am not sure why this happens and if I know by which this function merge two datasets, I will probably be able to solve this.  

Thank you
<img width="194" alt="화면 캡처 2022-05-20 234944" src="https://user-images.githubusercontent.com/77565848/169554187-96eac637-1371-43fe-9b0e-887c4b6a1655.png">
!

**Comment by WeilerP** [comment]
@parkjooyoung99, the respective code is [here](https://github.com/theislab/scvelo/blob/85295fc43e9582423e4b977d5fba5c2c65869d71/scvelo/core/_anndata.py#L502). My best guess is that the two AnnData objects you are trying to merge do not share any observation names.
